code/modules/mob/mob_movement.dm

```diff
@@ -137,8 +137,12 @@
 
 	if(isturf(mob.loc))
 
+
+		var/turf/T = mob.loc
 		move_delay = world.time//set move delay
 
+		move_delay += T.slowdown

```

Comment:
if mob.loc isn't a turf this will runtime.
You need var/turf/T = get_turf(mob) if(T) move_delay += T.slowdown